plugins/CorePluginsAdmin/vue/src/PasswordConfirmation/PasswordConfirmation.vue

```diff
@@ -8,9 +8,18 @@
   <div class="confirm-password-modal modal" ref="root">
     <div class="modal-content">
       <div class="modal-text">
-        <slot></slot>
+        <div ref="content"><slot></slot></div>
+        <h2 v-if="!slotHasContent && !requiresPasswordConfirmation">
```

Comment:
to keep the order of the conditions in the v-if same for all h2 variations
```suggestion
        <h2 v-if="!requiresPasswordConfirmation && !slotHasContent">
```